pkg/cmd/cli/backup/create.go

```diff
@@ -128,6 +135,17 @@ func (o *CreateOptions) Validate(c *cobra.Command, args []string, f client.Facto
 		return err
 	}
 
+	if o.FromSchedule != "" {
+		if (len(o.IncludeNamespaces) > 1 || o.IncludeNamespaces[0] != "*") ||
```

Comment:
This is going to be painful to maintain, is there a better way to check for this? Alternatively:

1. We ignore these flags and mention other filters will get ignored in the description for the `--from-schedule` flag
2. We allow users to override the Schedule with these flags

Reply:
yeah, this is not ideal. I'm leaning towards #1 for simplicity - WDYT?

Reply:
+1 agreed, I think just logging here that other filters are ignored is probably the best approach here